packages/wrangler/src/index.tsx

```diff
@@ -57,11 +57,19 @@ const knownBindings = [
   "durable_object_namespace",
 ];
 
+type ConfigPath = string | undefined;
+
+async function findWranglerToml(
+  referencePath: string = process.cwd()
+): Promise<ConfigPath> {
+  const configPath = await findUp("wrangler.toml", { cwd: referencePath });
+  return configPath;
+}
+
 async function readConfig(configPath?: string): Promise<Config> {
   const config: Config = {};
   if (!configPath) {
-    configPath = await findUp("wrangler.toml");
-    // TODO - terminate this early instead of going all the way to the root
+    configPath = await findWranglerToml(process.cwd());
   }
```

Comment:
How about always running `findWranglerToml()` from the path given, and make is so that it exits as soon as the path is actually a `wrangler.toml` file?

Reply:
Sorry I must not be understanding. `findWranglerToml` does stop looking once it finds a wrangler.toml.

Reply:
Sorry I am being confusing. Let me put together what I mean...